packages/expo-cli/src/credentials/credentialsJson/__tests__/update-test.ts

```diff
@@ -0,0 +1,293 @@
+      const keystore = await fs.readFile('./keystore.jks', 'base64');
+      const newCredJson = await fs.readJson('./credentials.json');
+      expect(keystore).toEqual(testKeystore.keystore);
+      expect(newCredJson).toEqual({
+        android: {
+          keystore: {
+            keystorePath: 'keystore.jks',
+            keystorePassword: testKeystore.keystorePassword,
+            keyAlias: testKeystore.keyAlias,
+            keyPassword: testKeystore.keyPassword,
+          },
+        },
+        ios: {
+          test: '123',
+        },
+      });
+    });
+  });
+  describe('updateIosCredentialsAsync', () => {
+    it('should update ios credentials if everything is correct', async () => {
+      const ctx = getCtxMock();
+      vol.fromJSON({
+        './credentials.json': JSON.stringify({
+          ios: {
+            provisioningProfilePath: 'pprofile',
+            distributionCertificate: {
+              path: 'cert.p12',
+              password: 'certPass',
+            },
+          },
+        }),
+        './pprofile': 'somebinarycontent',
+        './cert.p12': 'somebinarycontent2',
+      });
+      await credentialsJsonUpdateUtils.updateIosCredentialsAsync(ctx, 'bundleIdentifier');
+      const certP12 = await fs.readFile('./cert.p12', 'base64');
+      const pprofile = await fs.readFile('./pprofile', 'base64');
+      const credJson = await fs.readJson('./credentials.json');
+      expect(certP12).toEqual(testAllCredentialsForApp.distCredentials.certP12);
+      expect(pprofile).toEqual(testAllCredentialsForApp.credentials.provisioningProfile);
+      expect(credJson).toEqual({
+        ios: {
+          provisioningProfilePath: 'pprofile',
+          distributionCertificate: {
+            path: 'cert.p12',
+            password: testAllCredentialsForApp.distCredentials.certPassword,
+          },
+        },
+      });
+    });
+    it('should update ios credentials if credentials.json does not exists', async () => {
+      const ctx = getCtxMock();
+      await credentialsJsonUpdateUtils.updateIosCredentialsAsync(ctx, 'bundleIdentifier');
+      const certP12 = await fs.readFile('./ios/certs/dist-cert.p12', 'base64');
+      const pprofile = await fs.readFile('./ios/certs/profile.mobileprovision', 'base64');
+      const credJson = await fs.readJson('./credentials.json');
+      expect(certP12).toEqual(testAllCredentialsForApp.distCredentials.certP12);
+      expect(pprofile).toEqual(testAllCredentialsForApp.credentials.provisioningProfile);
+      expect(credJson).toEqual({
+        ios: {
+          provisioningProfilePath: './ios/certs/profile.mobileprovision',
+          distributionCertificate: {
+            path: './ios/certs/dist-cert.p12',
+            password: testAllCredentialsForApp.distCredentials.certPassword,
+          },
+        },
+      });
+    });
+    it('should not do anything if no credentials are present', async () => {
```

Comment:
`if no credentials are present` - present where?
Make the name more meaningful.